catt/controllers.py

```diff
@@ -337,14 +336,14 @@ def wait_for_states(self, timeout=None):
 
 
 class CastController:
-    def __init__(self, cast, name, app_id, prep=None):
+    def __init__(self, cast, app, prep=None):
```

Comment:
Can we add types here?

Reply:
Let's see if Miss Mypy has one of her good days.

Reply:
:crossed_fingers:

Reply:
Mypy doesn't like me running ```register_handler``` on an attribute that might not be there, so I made that error go away.

Reply:
Sounds good!